app/config/database.js

```diff
@@ -0,0 +1,43 @@
+import { DefaultAzureCredential } from '@azure/identity'
+import environments from '../constants/environments.js'
+
+const isProd = () => {
+  return process.env.NODE_ENV === environments.PRODUCTION
+}
+
+const hooks = {
+  beforeConnect: async (dbConfig) => {
```

Comment:
Renamed this `dbConfig` to prevent a sonarcloud clash with the config defined on line 27